services/app/src/static/style/loginScreen.css

```diff
@@ -0,0 +1,8 @@
+.login-screen {
+    background: #f3f6fa;
+    min-height: 100vh;
+    max-height: 100vh;
+    display: flex;
```

Comment:
This set of options 
```
display: flex;
flex-direction: column;
justify-content: space-between;
```
Could be extracted into a separate class in the `layout.css` file and reused.